mne/annotations.py

```diff
@@ -1098,6 +1098,16 @@ def annotations_from_events(events, sfreq, event_desc=None, first_samp=0,
     Notes
     -----
     Annotations returned by this function will all have zero (null) duration.
+
+    Creating events from annotations via the function
+    :func:`mne.events_from_annotations` takes in event mappings with
```

Comment:
```suggestion
    `mne.events_from_annotations` takes in event mappings with
```
nowadays sphinx can find things in the mne namespace without explicit `:func:`, `:class:` role declarations